regression/spec/personalInfo.spec.js

```diff
@@ -0,0 +1,99 @@
+const homepage = require('../pageObjects/homepage');
+const login = require('../pageObjects/login');
+const myAccount = require('../pageObjects/myAccount');
+
+describe('regression tests', ()=> {
+
+  beforeEach(async function() {  
+    browser.ignoreSynchronization = true;
+    await browser.get('http://ec2-3-124-113-90.eu-central-1.compute.amazonaws.com:8090/');
+    await browser.manage().window().maximize();
+  });
+
+  it('RT039',async function(){
+    await homepage.clickOn("MY ACCOUNT");
+    await login.insertEmail("adna.torlo@edu.fit.ba");
```

Comment:
you could define at beginning of email, password etc variable since you use it on multiple locations.